calico_node/filesystem/etc/rc.local

```diff
@@ -1,3 +1,8 @@
+# Log an info level log to stdout (JSON format)
+log() {
+	echo '{"msg":"progress: '$1'","severity":"info","time":"'$(date -Iseconds -u)'Z"}'
```

Comment:
Is json the right thing to emit? Shouldn't this mirror the output format we use for `log.Infof`?